Source/EasyNetQ.Serialization.NewtonsoftJson/Preconditions.cs

```diff
@@ -0,0 +1,74 @@
+using System;
+
+namespace EasyNetQ.Serialization.NewtonsoftJson
+{
+    /// <summary>
+    /// Collection of precondition methods for qualifying method arguments.
+    /// </summary>
+    internal static class Preconditions
```

Comment:
This file from EasyNetQ project could be referenced as is if use `link` attribute in csproj.

Reply:
Or maybe just get rid of this file and do argument validation directly in the methods, I don't think this adds much